pkg/converter/configuration/converter.go

```diff
@@ -264,6 +265,10 @@ func (pf *ProviderPkgFamilyParameters) ProviderPackageV1(p xppkgv1.Provider) ([]
 			continue
 		}
 		if extractProviderNameFromPackageName(p.Spec.PackageSpec.Package) == pf.Monolith {
+			cc := p.Spec.ControllerConfigReference
+			controllerConfigReference := xppkgv1.ControllerConfigReference{
```

Comment:
We may consider adding a nil check here.

Reply:
done @sergenyalcin